src/Identity/ApiAuthorization.IdentityServer/src/Microsoft.AspNetCore.ApiAuthorization.IdentityServer.csproj

```diff
@@ -8,19 +8,29 @@
     <!-- We are a package that depends on the shared framework, this allows us to
          avoid errors during restore -->
     <GenerateErrorForMissingTargetingPacks>false</GenerateErrorForMissingTargetingPacks>
+    <!-- IdentityServer packages are not signed -->
+    <NoWarn>CS8002</NoWarn>
     <Nullable>disable</Nullable>
+    <!-- REMOVE THIS -->
+    <AddPublicApiAnalyzers>false</AddPublicApiAnalyzers>
   </PropertyGroup>
 
   <ItemGroup>
     <!-- Remove the reference to Microsoft.AspNetCore.App from IdentityServer4 -->
     <FrameworkReference Remove="Microsoft.AspNetCore.App" />
-    <Reference Include="IdentityServer4.AspNetIdentity" />
-    <Reference Include="IdentityServer4.EntityFramework" />
-    <Reference Include="IdentityServer4.EntityFramework.Storage" />
-    <Reference Include="IdentityServer4" />
-    <Reference Include="IdentityServer4.Storage" />
+    <Reference Include="Duende.IdentityServer.AspNetIdentity" />
+    <Reference Include="Duende.IdentityServer.EntityFramework" />
+    <Reference Include="Duende.IdentityServer.EntityFramework.Storage" />
+    <Reference Include="Duende.IdentityServer" />
+    <Reference Include="Duende.IdentityServer.Storage" />
     <Reference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" />
     <Reference Include="Microsoft.AspNetCore.Identity.UI" />
+    <Reference Include="Newtonsoft.Json" />
+    <SuppressBaselineReference Include="IdentityServer4.AspNetIdentity" />
```

Comment:
@aspnet-build is this something I should be checking in (SupressBaselineReference?)

Reply:
It's alright if the pseudo-breaking change is approved. I'm a bit more concerned about having a shipping package depend on an unsigned assembly. Thoughts @Pilchie❔

Reply:
This would be my only concern as well. I couldn't find other instances of shipping projects doing this in our repo or dotnet/runtime

Reply:
@blowdart seems to be okay with this, is there anyone else we should ping about this?

Reply:
Our Nuget package is signed. The assemblies do not have a strong name. What would be the benefit of that for a library targeting .NET Core consumers only?